### PR TITLE
feat: add migrate from deploy guide.

### PR DESCRIPTION
**Pull request by raksiv**
This is an unlisted page - so not added to links.

### CODE REVIEW

File: src/pages/guides/getting-started/migrating-from-deploy.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+```bash
+# Pulumi.yaml
+echo "name: cold-pond" > Pulumi.yaml
+echo "runtime: nodejs" >> Pulumi.yaml
+
+# Pulumi.cold-pond-aws.yaml
+pulumi stack select usr/cold-pond/cold-pond-aws
+pulumi config refresh
+```
```
Comment (tjholm) [inline src/pages/guides/getting-started/migrating-from-deploy.md:44]: I think we'd need to provide the current stack state to users so they can do this.

Comment (raksiv) [reply src/pages/guides/getting-started/migrating-from-deploy.md:44]: Aren't Pulumi steps in their stack is how you gain access to current state?

File: src/pages/guides/getting-started/migrating-from-deploy.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+export const description = 'Migrating your application away from Nitric Deploy'
+
+# Deploying with Nitric Deploy
+
+<Note>
+The public preview phase for Nitric Deploy has concluded, and we're no longer accepting sign-ups at this time. We are incredibly thankful for the support and feedback we've received from our preview participants. This feedback has been invaluable in helping us improve both Nitric Deploy and the open-source Nitric Framework.
+
+We're hard at work on the future of the open-source Nitric Framework and other services from the Nitric team. We appreciate your interest and encourage you to follow our latest announcements elsewhere on this site, [GitHub](https://github.com/nitrictech/nitric) and [Discord](https://discord.gg/Webemece5C).
```
Comment (davemooreuws) [inline src/pages/guides/getting-started/migrating-from-deploy.md:8]:
```suggestion
We're hard at work on the future of the open-source Nitric Framework and other services from the Nitric team. We appreciate your interest and encourage you to follow our latest announcements elsewhere on this site, [GitHub](https://github.com/nitrictech/nitric) and [Discord](https://nitric.io/chat).
```

File: src/pages/guides/getting-started/migrating-from-deploy.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+- Set up an actionable event triggered by a 'git push' to your desired branch
+- Nominate an runtime OS for your CI
+- Install code dependencies and build the application
+- Install Nitric and its dependencies.
+- Configure environment variables
+
+Click [here](https://nitric.io/docs/guides/getting-started/github-actions) for a guide on setting up a GitHub Actions.
+
+> Note: Please reach out on [Discord](https://discord.gg/Webemece5C) if you encounter any difficulties or if you would like instructions on migrating to other CI solutions.
```
Comment (davemooreuws) [inline src/pages/guides/getting-started/migrating-from-deploy.md:28]:
```suggestion
> Note: Please reach out on [Discord](https://nitric.io/chat) if you encounter any difficulties or if you would like instructions on migrating to other CI solutions.
```

File: src/pages/guides/getting-started/migrating-from-deploy.md (hunk 1 above)
Comment (davemooreuws) [reply src/pages/guides/getting-started/migrating-from-deploy.md:44]: Would the refresh flag help here?

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Pull request closed without merging**